creator/ingest_runs/queries/validation_run.py

```diff
@@ -0,0 +1,37 @@
+from graphene import relay
+from graphene_django.filter import DjangoFilterConnectionField
+from graphql import GraphQLError
+from django_filters import FilterSet, OrderingFilter
+
+from creator.ingest_runs.nodes import ValidationRunNode
+from creator.ingest_runs.models import ValidationRun
+
+
+class ValidationRunFilter(FilterSet):
```

Comment:
Do we ever need to filter by study or data review?

Reply:
Maybeee? If we had a view that listed _all_ validation runs then these two filters would be useful. But right now all we have is a detail view for a data review where would likely display a list of validation runs for that review by accessing `DataReview.validation_runs`.